Gruntfile.js

```diff
@@ -745,7 +745,7 @@ module.exports = function (grunt) {
             dev: {
                 expand: true,
                 flatten: false,
-                src: ["<%= yeoman.client %>/{app,components}/**/translations/*fr_FR.xml"],
+                src: ["<%= yeoman.client %>/{app,components}/**/translations/*.xml"],
```

Comment:
Maybe you should just take fr_FR and en_GB instead of all translations. We want to keep dev environment as fast as possible.

Reply:
Agreed. try: ["<%= yeoman.client %>/{app,components}/**/translations/*{fr_FR, en_GB}.xml"].

Reply:
+1